### PR TITLE
✨ Add stats bar, icons

### PR DESCRIPTION
**Pull request by anncatton**
**Description of changes**

Adds StatsBar component to File Repository
- displayed with mock data
- new icons: `crosshairs`, `filesize`

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/file-repository/StatItem/index.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { capitalize } from 'global/utils/stringUtils';
+
+type TSTatType = 'file' | 'primary site' | 'donor' | 'program' | 'filesize';
+type StatItemProps = {
+  iconName: UikitIconNames;
+  statType: TSTatType;
+  count: number;
+};
+
+// for mock data display, not sure how this will be calculated esp for file size
```
Comment (wajiha-oicr) [inline components/pages/file-repository/StatItem/index.tsx:16]: I'm not entirely sure either how the file size data is going to look like when it comes in, but when formatting that in the donor table, I based it under the assumption the information would arrive in bytes ( this makes sorting easier :P ). Then I imported and used `fileSize` which automatically handles formatting the number with the correct unit (i.e kb, mb, tb etc) . 


https://www.npmjs.com/package/filesize
https://github.com/icgc-argo/platform-ui/blob/develop/components/pages/file-repository/FileTable/index.tsx#L107

Comment (anncatton) [reply components/pages/file-repository/StatItem/index.tsx:16]: Cool! Yeah I wasn't sure either, so i put some oversimplified math

File: components/pages/file-repository/StatsCard/StatItem.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import filesize from 'filesize';
+import { css } from 'uikit';
+import { UikitIconNames } from 'uikit/Icon/icons';
+import { capitalize } from 'global/utils/stringUtils';
+import { Col } from 'react-grid-system';
+import { useTheme } from 'uikit/ThemeProvider';
+import Icon from 'uikit/Icon';
+import Typography from 'uikit/Typography';
+
+type TSTatType = 'file' | 'primary site' | 'donor' | 'program' | 'filesize';
```
Comment (wajiha-oicr) [inline components/pages/file-repository/StatsCard/StatItem.tsx:10]: Just curious about the naming convention here? What does TSTatType mean?

File: components/pages/file-repository/StatsCard/StatItem.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+type StatItemProps = {
+  iconName: UikitIconNames;
+  statType: TSTatType;
+  count: number;
+};
+
+// for mock data display. this could change when data is integrated
+const getDisplayStat = (type: TSTatType, count: number): string => {
+  return type === 'filesize' ? `${filesize(count)}` : `${count} ${capitalize(type)}s`;
```
Comment (wajiha-oicr) [inline components/pages/file-repository/StatsCard/StatItem.tsx:20]: 👍👍👍

Also, adding 's' works but if there's ever a case where there's only 1 of them then the grammar doesn't make sense. I think `pluralize` can take care of the grammar rule in these cases. Not needed for now, but something that may come up later.

File: components/pages/file-repository/StatsCard/StatItem.tsx (hunk 2 above)
Comment (hlminh2000) [reply components/pages/file-repository/StatsCard/StatItem.tsx:10]: there is a naming strategy where you prefix the type of the thing with a shorthand for that type, so `T_StatType` to denote a type, `I_StatType` to denote an interface, `C_StatType` to denote a class...
Kinda an old school convention that's more helpful if we were using raw vim in my opinion :P

For us I think it's more noisy than helpful so wouldn't recommend unless there's a very tricky namespace collision going on, even then would say it's a last resort :P

Comment (anncatton) [reply components/pages/file-repository/StatsCard/StatItem.tsx:10]: Also noticed I had an extra capital `T` in there so will change that too :)

Comment (hlminh2000) [reply components/pages/file-repository/StatsCard/StatItem.tsx:10]: lol I totally thought you were going for the type prefix :P

Comment (anncatton) [reply components/pages/file-repository/StatsCard/StatItem.tsx:10]: oh yeah the first `T` was that for sure...but then I capitalized the next one too cuz I can't type :)

File: components/pages/file-repository/StatsCard/StatItem.tsx (hunk 3 above)
Comment (anncatton) [reply components/pages/file-repository/StatsCard/StatItem.tsx:20]: Yeah, I don't think the case will come up, but i'll add it in for safety :)

File: components/pages/file-repository/StatsCard/StatItem.tsx (hunk 2 above)
Comment (hlminh2000) [reply components/pages/file-repository/StatsCard/StatItem.tsx:10]: lol didn't even notice that :P like Jon says, I can't read...

**Pull request merged**